From a24c43567a3c94939f3cda145a74d7dc8c65f864 Mon Sep 17 00:00:00 2001
From: Adrian Alonso <adrian.alonso@nxp.com>
Date: Wed, 10 Aug 2016 14:43:13 -0500
Subject: [PATCH 1148/5242] MLK-13183: ARM: dts: imx6q pop arm2 set soc pu as
 shared

commit  0be9afc5f512132a7f2019479a9e0768587aba89 from
https://source.codeaurora.org/external/imx/linux-imx.git

Set soc arm as shared, arm_reg and soc_reg share the same
voltage rail source on imx6q pop arm2 target platform

Signed-off-by: Adrian Alonso <adrian.alonso@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6q-pop-arm2.dts |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx6q-pop-arm2.dts b/arch/arm/boot/dts/imx6q-pop-arm2.dts
index 3d0a23f..764e6a6 100644
--- a/arch/arm/boot/dts/imx6q-pop-arm2.dts
+++ b/arch/arm/boot/dts/imx6q-pop-arm2.dts
@@ -119,6 +119,10 @@
 	};
 };
 
+&cpu0 {
+	fsl,arm-soc-shared = <1>;
+};
+
 &busfreq {
 	fsl,max_ddr_freq = <400000000>;
 };
-- 
1.7.9.5

